sklearn/datasets/_samples_generator.py

```diff
@@ -342,6 +342,15 @@ def make_multilabel_classification(n_samples=100, n_features=20, n_classes=5,
         Only returned if ``return_distributions=True``.
 
     """
+    # Validation of the arguments
+    if n_classes == 0 and not allow_unlabeled:
```

Comment:
Actually we don't care about `allow_unlabeled`. We just should not create a classification dataset without any class.

```suggestion
    if n_classes < 1:
```